.circleci/config.yml

```diff
@@ -253,7 +239,6 @@ workflows:
       - test_unit
       - test_unit_react_v17
       - test_unit_algoliasearch_v3
-      - test_integration
```

Comment:
(unrelated to this specific line but will allow us to have a threaded discussion)

Should we remove the scripts in `package.json` as well as any lingering dependencies (argos-cli for example)?

Reply:
oh good one, I didn't think of that